doc/api/nmo-import-mongo.md

```diff
@@ -0,0 +1,11 @@
+nmo-import-mongo(3) -- import-mongo
+==============================
+
+## SYNOPSIS
+
+    nmo.commands.import-mongo([<url> || <cluster>], [database], [mongourl], [collection])
```

Comment:
still needs to get fixed `<` instead of `[` for database, mongourl and collection